src/System.Net.Primitives/tests/FunctionalTests/System.Net.Primitives.Functional.Tests.csproj

```diff
@@ -33,13 +33,14 @@
     <Compile Include="IPAddressMappingTest.cs" />
     <Compile Include="IPEndPointTest.cs" />
     <Compile Include="NetworkCredentialTest.cs" />
-    <Compile Include="RequestCachePolicyTest.cs" Condition="'$(TargetGroup)' == ''" />
     <Compile Include="SocketAddressTest.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == ''">
     <Compile Include="LoggingTest.cs" />
     <Compile Include="SerializationTest.cs" />
+    <Compile Include="RequestCachePolicyTest.cs" />
     <Compile Include="CookieContainerAddTest.cs" />
+    <Compile Include="NetworkCredentialTest.netstandard1.7.cs" />
```

Comment:
Sorry I meant only for netstandard1.7 is being set as ' '. But for netstandard1.3 would include the files added in the `<ItemGroup>` with no condition, and the original file is being added into that `<ItemGroup>`. So if I add my tests to the original file I would get the following compile error: 

```
NetworkCredentialTest.cs(30,75): error CS1061: 'NetworkCredential' does not contain a definition for 'SecurePassword' and no extension method 'SecurePassword' accepting a first argument of typ'NetworkCredential' could be found (are you missing a using directive or an assembly reference?) [C:\Users\safern\corefx\corefx\src\System.Net.Primitives\tests\FunctionalTests\System.Net.Primitives.Functional.Tests.csproj]
```

Where `SecurePassword` is the API I'm porting. @CIPop